lib/models/header.php

```diff
@@ -15,7 +15,7 @@ public function __construct() {
 
   public function getLogo() {
     if ($this->logo) {
-      return $this->logo['sizes']['thumbnail'];
+      return $this->logo['sizes']['large'];
```

Comment:
Will WordPress give us an overscaled logo if we ask for the large one? For exaple, if we upload a `120x80` or something, would it blow it up to something like `1280x800`?

We could use `$this->logo['url']` to get the original file upload.

Added https://github.com/SandersForPresident/WordPress/issues/6 to introduce custom sizes, such that we can later do `$this->logo['sizes']['logo']`